src/components/Search/search-result.jsx

```diff
@@ -1,104 +1,94 @@
 import { Link } from 'gatsby';
 import { default as React } from 'react';
-import { Paragraph } from '@smallstep/step-ui';
+import { Heading } from '@smallstep/step-ui';
 import { makeStyles } from '@material-ui/styles';
 import {
   connectStateResults,
+  connectHits,
   Highlight,
-  Hits,
   Index,
   Snippet,
 } from 'react-instantsearch-dom';
-import { Box, ListItem, ListItemText } from '@material-ui/core';
+import { Box, Typography, ListItem, ListItemText } from '@material-ui/core';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
```

Comment:
`makeStyles` accepts a function with `theme` as an argument, so we can use variables from our step-ui theme.